src/ReactiveUI.Validation.AndroidSupport/ReactiveUI.Validation.AndroidSupport.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+
+  <PropertyGroup>
+    <TargetFramework>MonoAndroid90</TargetFramework>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <Nullable>enable</Nullable>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Xamarin.Android.Support.Design" Version="28.*" />
+    <PackageReference Include="ReactiveUI" Version="11.*" />
```

Comment:
Where's this Android.Support.Design coming from? We have a ReactiveUI.AndroidSupport.

Reply:
`Xamarin.Android.Support.Design` is also required for the `TextInputLayout` control used in [`BindValidation`](https://github.com/reactiveui/ReactiveUI.Validation/blob/androidx/src/ReactiveUI.Validation.AndroidSupport/Extensions/ViewForExtensions.cs#L47). And this also doesn't seem to be included into `ReactiveUI.AndroidSupport` as we aren't referencing `Support.Design` packages there https://github.com/reactiveui/ReactiveUI/blob/main/src/ReactiveUI.AndroidSupport/ReactiveUI.AndroidSupport.csproj#L12